### PR TITLE
enhancing okta 2fa messaging include okta push

### PR DESCRIPTION
**Pull request by cviecco**

### CODE REVIEW

File: cmd/keymasterd/templateData.go [hunk 1]
```diff
@@ -176,6 +176,10 @@ const secondFactorAuthFormText = `
         {{if .ShowOktaOTP}}
 	<div id="okta_login_destination" style="display: none;">{{.LoginDestination}}</div>
         <form enctype="application/x-www-form-urlencoded" action="/api/v0/okta2FAAuth" method="post">
+	    <p>
+	    Okta push has been automatically started. If you are not able to recieve the
+	    push notification you can proceed by etnering the Okta OTP code.
+	    </p>
```
Comment (kyleip) [inline cmd/keymasterd/templateData.go:182]: nit: not aligned with the section below

**Review by kyleip — APPROVED** [review]
nit: not aligned with the section below :)

File: cmd/keymasterd/templateData.go [hunk 2]
```diff
@@ -176,6 +176,10 @@ const secondFactorAuthFormText = `
         {{if .ShowOktaOTP}}
 	<div id="okta_login_destination" style="display: none;">{{.LoginDestination}}</div>
         <form enctype="application/x-www-form-urlencoded" action="/api/v0/okta2FAAuth" method="post">
+            <p>
+            Okta push has been automatically started. If you are not able to recieve the
+            push notification you can proceed by etnering the Okta OTP code.
```
Comment (erikespinoza) [inline cmd/keymasterd/templateData.go:181]: entering, not etnering

File: cmd/keymasterd/templateData.go [hunk 3]
```diff
@@ -176,6 +176,10 @@ const secondFactorAuthFormText = `
         {{if .ShowOktaOTP}}
 	<div id="okta_login_destination" style="display: none;">{{.LoginDestination}}</div>
         <form enctype="application/x-www-form-urlencoded" action="/api/v0/okta2FAAuth" method="post">
+            <p>
+            Okta push has been automatically started. If you are not able to recieve the
```
Comment (erikespinoza) [inline cmd/keymasterd/templateData.go:180]: receive, not recieve

File: cmd/keymasterd/templateData.go (hunk 1 above)
Comment (cviecco) [reply cmd/keymasterd/templateData.go]: solved

File: cmd/keymasterd/templateData.go (hunk 2 above)
Comment (cviecco) [reply cmd/keymasterd/templateData.go]: thanks for the typo corrections. Addressed

**Review by erikespinoza — APPROVED** [review]

**Pull request merged**